lib/ansible/config/base.yml

```diff
@@ -1311,6 +1311,7 @@ CONNECTION_FACTS_MODULES:
     junos: junos_facts
     nxos: nxos_facts
     vyos: vyos_facts
+    exos: exos_facts
```

Comment:
@trishnaguha Nice work on your presentation - 'Deep Dive into Ansible Network Resource Module', very resourceful!

During the talk, you mentioned that `gather_facts: True` has been fixed to work for network devices by gathering facts about the network device and not the local control node. So, I thought you might have an idea about how it was achieved. I believe this mapping(ANSIBLE_CONNECTION_FACTS_MODULES) of NOS to NOS_FACTS module is what let the 'gather_facts' work for network devices. Is that right? 

I couldn't get it to work for EXOS, would appreciate your help!

Reply:
You know what, I got it working now! I'm not sure what I was missing. Except, I pulled in all the latest changes.